standard/lua.lua

```diff
@@ -9,6 +9,20 @@
 local FeatureFlag = require('Module:FeatureFlag')
 local Logic = require('Module:Logic')
 local StringUtils = require('Module:StringUtils')
+local Table = require('Module:Table')
+
+local LOADABLE_LIBRARIES = Table.map({
+	'bit32',
+	'libraryUtil',
+	'luabit.bit',
+	'luabit.hex',
+	'strict',
+	'ustring', -- Use mw.ustring instead!
```

Comment:
Are these actually used?

Reply:
I think there might be some usage of string and lib util and strict in some old code. Not sure of the rest. Supporting them incase someone gas a need for them